clients/kau-account-usage/src/kau-reports/services/kau-reports-data.service.js

```diff
@@ -43,7 +43,7 @@ module.exports = function($q, kaKalturaAPIFacade, kauReportsConfiguration)
 
             var requestParams = {
                 reportType: filters.reportType,
-                reportInputFilter: {fromDay: moment(filters.date.startDate).format('YYYYMMDD'), toDay: moment(filters.date.endDate).format('YYYYMMDD')}
+                reportInputFilter: {interval: filters.interval ,fromDay: moment(filters.date.startDate).format('YYYYMMDD'), toDay: moment(filters.date.endDate).format('YYYYMMDD')}
```

Comment:
did you check that it doesn't break the existing reports that doesn't have interval?

Reply:
yes works ok